### PR TITLE
Adding screen infos and setting default brightness

### PR DESCRIPTION
**Pull request by mauronofrio**

### CODE REVIEW

**Comment by notsyncing** [comment]
Mine is working without these configs, and I'm able to adjust brightness. Are they necessary?

**Comment by simonsmh** [comment]
[Actually not.](https://github.com/omnirom/android_bootable_recovery/blob/cbd8cc5e06656a40f45b78adbb988b59564cb17a/data.cpp#L800)

**Review by simonsmh — APPROVED** [review]

**Comment by mauronofrio** [comment]
It's to set the initial brightness...

**Comment by notsyncing** [comment]
Then I guess we only need the `TW_DEFAULT_BRIGHTNESS := 1500` line only.

File: BoardConfig.mk [hunk 1]
```diff
@@ -99,6 +99,9 @@ RECOVERY_SDCARD_ON_DATA := true
 BOARD_HAS_NO_REAL_SDCARD := true
 TARGET_RECOVERY_QCOM_RTC_FIX := true
 TARGET_RECOVERY_PIXEL_FORMAT := "BGRA_8888"
+TW_BRIGHTNESS_PATH := "/sys/class/backlight/panel0-backlight/brightness"
```
Comment (notsyncing) [inline BoardConfig.mk:102]: Not needed.

File: BoardConfig.mk [hunk 2]
```diff
@@ -99,6 +99,9 @@ RECOVERY_SDCARD_ON_DATA := true
...
+TW_MAX_BRIGHTNESS := 4095
```
Comment (notsyncing) [inline BoardConfig.mk:103]: Not needed.

**Review by notsyncing — CHANGES_REQUESTED** [review]

**Comment by mauronofrio** [comment]
It's better if you set all 3 flags or you.can have some problems then with brightness, the twrp is able to read automatically the brightness path but not the max value.

**Comment by notsyncing** [comment]
Okay.

**Pull request merged**